### DISCUSSION TITLE
New MacOS Update.

### DISCUSSION THREAD

**Discussion by dajahsweet** [inkstitch/inkstitch#2640] [General]
So I was looking to update inkscape/inkstitch but I recently updated my Mac Air. The new update for my Mac is called macOS Sonoma. My only concern is that if I update Inkscape will it and the new features for MacOS still work. I haven't used Inkscape in a while and haven't updated to either 3.0.0 or the 3.0.1 from June.

**Comment by kaalleen** [comment]
It should work on Sonoma
